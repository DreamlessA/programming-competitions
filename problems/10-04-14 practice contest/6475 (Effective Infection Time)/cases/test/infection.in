11
2 0009
11 0012
3 0010
10 0010
2 0012
11 0012
2 0009
2 0009
12 0009
1 0010
1 0000
1 0000
1 0000
2 0000
3 0000
8 0001
7 0007
7 0007
3 0005
12 0015
9 0005
9 0007
