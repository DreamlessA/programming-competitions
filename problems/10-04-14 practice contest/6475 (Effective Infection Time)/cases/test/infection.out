3.3333
0.3500
0.4091
0.0000
0.5000
0.0000
0.0417
1.0833
0.0000
10.4167
2.1667
